plugins/disassemble/disassemble_main_rules.ml

```diff
@@ -172,13 +172,15 @@ module Symbols = struct
           })
       ~unmatched:(fun reason t -> match reason with
           | Non_injective_fwd (addrs,name) ->
-            info "the symbol %s has ambiguous addresses: %a@\n"
+            info "the symbol %s has ambiguous addresses: %a@"
```

Comment:
`@\n` is Format's new line, why did you remove `\n`? If you think that a new line is unnecessary here from the aesthetic perspective, then please remove the `@` as well.